services/hooks/src/v1.js

```diff
@@ -494,9 +495,9 @@ builder.declare({
 /**
  * Common implementation of triggerHook and triggerHookWithToken
  */
-const triggerHookCommon = async function({req, res, hook, payload, firedBy}) {
+const triggerHookCommon = async function({req, res, hook, payload, clientId = null, firedBy}) {
```

Comment:
this will automatically default to `undefined` (which is what we want -- the context item shouldn't be set at all, rather than being set to null), so no need for the `= null` here.